client/js/components/tags/TagListBulkControls.vue

```diff
@@ -0,0 +1,26 @@
+<template>
+  <div class="mb">
+    <label class="mb mr-[18px] inline-block">
+      <input
+        class="mr-0"
+        type="checkbox"
+        data-cy="checkAllItems"
+        name="checkAllItems">
+      {{ Translator.trans('all') }}
+    </label>
+
+    <button
+      class="btn-icns ml"
+      name="r_delete"
+      data-cy="deleteSelectedItems"
```

Comment:
should we be more specific with the naming of `data-cy`, smth like `bulkControls:deleteSelectedItems`